pygsti/algorithms/fiducialpairreduction.py

```diff
@@ -680,97 +945,374 @@ def _get_number_amplified(m0, m1, len0, len1):
 
 
 # Helper function for per_germ and per_germ_power FPR
-def _get_per_germ_fidpairs(prep_fiducials, meas_fiducials, pre_povm_tuples,
-                           gsGerm, mem_limit, printer, search_mode, seed, n_random):
+def _get_per_germ_power_fidpairs(prep_fiducials, meas_fiducials, pre_povm_tuples,
+                                 gsGerm, power, mem_limit, printer, search_mode, seed, n_random, dof_per_povm, 
+                                 min_iterations=1, lowest_eigenval_tol=1e-1,
+                                 candidate_set_seed=None, num_soln_returned=1, type_soln_returned='best'):
     #Get dP-matrix for full set of fiducials, where
     # P_ij = <E_i|germ^exp|rho_j>, i = composite EVec & fiducial index,
     #   j is similar, and derivs are wrt the "eigenvalues" of the germ
     #  (i.e. the parameters of the gsGerm model).
+    
+    #debugging
+    print('Entered helper function _get_per_germ_power_fidpairs')
```

Comment:
Maybe comment out these debug statements? In this whole function actually

Reply:
Some of these are debug statements and others are for progress logging. I'll go ahead and comment out the debug ones and swap the progress logging with calls to one of pygsti's VerbosityPrinter objects with an appropriate verbosity flag so they only show up when desired.

Reply:
I've made the aforementioned logging changes.